modules/KalturaSupport/resources/webcast/cuePointsManager.js

```diff
@@ -0,0 +1,501 @@
+(function (mw, $) {
```

Comment:
This flle was moved from the dualscreen plugin area to a shared location. @amirch1 instructed us to use the resources folder and add 'webcast' as a namespace so our components will not interfer to other plugins